### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Vistaus**
🙂

### CODE REVIEW

File: po/nl.po [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+msgid "Logout"
+msgstr "Uitloggen"
+
+#: ../qml/MainPostsPage.qml:93 ../qml/SearchPostsPage.qml:66
+msgid "Select sorting method"
+msgstr "Kies een sorteermethode"
+
+#: ../qml/MoreCommentLook.qml:46
+msgid "more comments..."
+msgstr "Meer reacties…"
```
Comment (Dennisbonke) [inline po/nl.po:60]: This shouldn't start with an uppercase letter.

File: po/nl.po [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+#: ../qml/OpenedRedditPost.qml:139
+msgid "comments:"
+msgstr "Reacties:"
```
Comment (Dennisbonke) [inline po/nl.po:64]: Same here, shouldn't start with an uppercase letter.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
Two small things but as a Dutchie, I approve of this translation.

File: po/nl.po (hunk 1 above)
Comment (Vistaus) [reply po/nl.po:60]: As a fellow Dutchie, I thought it made more sense in both cases, so I'll wait for the dev to respond. If he also thinks it should be lowercase, I'll change it, but thanks for sharing your opinion! 🙂

Comment (ElectrodeYT) [reply po/nl.po:60]: Thanks for the contribution!
Yea, it should be lower-case here.

Comment (Vistaus) [reply po/nl.po:60]: Alright, I'll change it right away!

Comment (Vistaus) [reply po/nl.po]: Done!

**Comment by ElectrodeYT** [comment]
Thanks!

**Pull request merged**